frontend/src/helpers/utils.ts

```diff
@@ -82,3 +83,31 @@ export function parseJSONsafe(presumedJSON: string): any | null {
 export function formatDateToIsoString(dateToIsoString: Date) {
   return format(dateToIsoString, "yyyy-MM-dd");
 }
+export function calculatePrice(arrayItems: Pick<OfferItem, "amount" | "price">[], date?: string) {
+  let amountNet = 0;
+  let amountGross = 0;
+  let amountVat = 0;
+
+  for (let i = 0; i < arrayItems.length; i++) {
+    const amount = arrayItems[i].amount ?? 0;
+    const price = arrayItems[i].price ?? 0;
+
+    amountNet += round(amount * price, 2);
+    amountGross += round(amountNet * (1 + getVatRate({ isoDate: date })), 2);
+    amountVat = round(amountGross - amountNet, 2);
+  }
+
+  return {
+    calculatedResultNetto: formatNumber(amountNet, { decimals: 2, currency: true }),
+    calculatedResultBrutto: formatNumber(amountGross, { decimals: 2, currency: true }),
+    calculatedResultUst: formatNumber(amountVat, { decimals: 2, currency: true }),
```

Comment:
Let's try to use English words everywhere except the UI

```suggestion
    amountNet: formatNumber(amountNet, { decimals: 2, currency: true }),
    amountGross: formatNumber(amountGross, { decimals: 2, currency: true }),
    amountVat: formatNumber(amountVat, { decimals: 2, currency: true }),
```